osmtm/views/project.py

```diff
@@ -100,6 +100,34 @@ def project(request):
                 priority_areas=FeatureCollection(features),)
 
 
+@view_config(route_name="project_json", renderer='json',
+             permission="project_show",
+             http_cache=0)
+def project_json(request):
+    id = request.matchdict['project']
+    request.response.content_disposition = \
+        'attachment; filename="hot_osmtm_%s.json"' % id
```

Comment:
I would avoid putting HOT in the file name. The tasking manager can be used by other organizations.